src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx

```diff
@@ -50,30 +29,31 @@ function getLocation(
 }
 
 export default function VideoVisitLocation({ header, appointment, facility }) {
-  const videoKind = getVideoKind(appointment);
-  const isAtlas = isAtlasLocation(appointment);
+  const { kind, isAtlas, providers } = appointment.videoData;
   const isHome = isVideoHome(appointment);
-  const isVideo = isVideoAppointment(appointment);
-  const isGFE = isVideoGFE(appointment);
   const [showMoreOpen, setShowMoreOpen] = useState(false);
   const phone = facility?.telecom?.find(tele => tele.system === 'phone')?.value;
   const name = facility?.name;
 
-  const isCommunityCare = appointment.vaos.isCommunityCare;
   const location = getLocation(
     isAtlas,
-    isVideo,
-    videoKind,
-    isCommunityCare,
+    appointment.vaos.isVideo,
+    kind,
     facility,
     appointment,
   );
 
-  if (appointment.vaos.isPastAppointment && videoKind === VIDEO_TYPES.clinic) {
+  const showVideoInstructions =
```

Comment:
Probably a good candidate for a helper, I think a follow up ticket to go through the refresh detail pages looking for logic to extract into helpers would make sense.